dev-games/tiled/metadata.xml

```diff
@@ -1,8 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!-- maintainer-needed -->
+	<maintainer type="person" proxied="yes">
+		<email>mechakotik@gmail.com</email>
+		<name>Andrei Sabalenka</name>
+	</maintainer>
+	<maintainer type="project" proxied="proxy">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
 	<upstream>
 		<remote-id type="github">bjorn/tiled</remote-id>
 	</upstream>
+	<use>
+		<flag name="headers">Install development headers</flag>
```

Comment:
Gentoo usually chooses to unconditionally install headers and such. Is there reason to make it conditional?

Reply:
Most Tiled users don't need these headers, why should ebuild force installing them?

Reply:
https://devmanual.gentoo.org/general-concepts/use-flags/index.html#when-not-to-use-use-flags

>  USE flags must not be used to control installing files that are small, non-intrusive, do not introduce additional build-time dependencies or cause a significant increase in build time. Examples of such files are bash completion files, init.d scripts, logrotate configuration files, systemd service files. The rationale is the same as above. Instead, those files must be installed unconditionally.